addons/l10n_in_website_sale/controllers/portal.py

```diff
@@ -0,0 +1,59 @@
+from odoo.addons.account.controllers.portal import CustomerPortal
+from odoo.http import request
+from odoo import _
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class L10nInCustomerPortal(CustomerPortal):
+
+    def _prepare_portal_layout_values(self):
+        portal_layout_values = super()._prepare_portal_layout_values()
+        portal_layout_values['l10n_in_gst_treatments'] = {
+            'regular': 'Registered Business - Regular',
```

Comment:
Just to be consistent you can create const in `l10n_in`, and also use the same in the selections in 
`odoo/addons/l10n_in/models/account_invoice.py`